app/templates/_service_attributes.html

```diff
@@ -2,21 +2,23 @@
 
 {% for section in service.summary_manifest %}
   {% if not section.is_empty %}
-    {{ summary.heading(section.name, id=section.slug) }}
+    <div class="scrollTracking" id="{{service.lot.slug}}-{{section.slug}}">
```

Comment:
Looks like we normally use kebab-case for classes and ids, so would be good to keep that consistency (i.e. `scroll-tracking`).

Also we normally pad either side of `{{` and `}}` with a space. Since you're here, would be nice to do that as well (i.e. `{{ service.lot.slug }}-{{ section.slug }}`